P5-51: Removed we and our from report (#149)
aau-hcl-p5/main: report/6.Conclusion/development_practices.tex

\subsection{Scrum}
The cooperation practices were heavily inspired by agile development, as it is predictable that several issues in a research-based project will arise during the period of the project.
However, rigorously following the Scrum principles was deemed unfeasible, as the project had a strict deadline, and a lack of an actual customer who could function as product owner.

> With this in mind, the principles from agile development, and especially Scrum, that were suitable were used instead of strictly following one specific model.

Reviewer: ```suggestion
With this in mind, principles from agile development, and especially Scrum, were used instead of strictly following one specific model, whenever applicable.
```
